daemon/api.go

```diff
@@ -300,7 +303,7 @@ func loginUser(c *Command, r *http.Request, user *auth.UserState) Response {
 	overlord := c.d.overlord
 	state := overlord.State()
 	state.Lock()
-	_, err = auth.NewUser(state, loginData.Username, macaroon, []string{discharge})
+	_, err = auth.NewUser(state, loginData.Username, macaroon, []string{discharge}, macaroon, []string{discharge})
```

Comment:
To be sure, when a local user logs in (against the store), are we creating a new user?